site2/website/contributing.md

```diff
@@ -295,6 +295,27 @@ the static content is generated by CI job and merged into the `asf-site` branch.
 
 Follow the [README](https://github.com/apache/pulsar/tree/master/site2) for making contributions to the website.
 
+#### Updating user logo
+The Apache Pulsar Community welcomes all users to update their company logos on Pulsar website. If your team or organization has adopted Pulsar in production, please update your company logos on [Pulsar website](https://pulsar.apache.org/en/powered-by/).
+
+It only takes 4 steps to update your company logo:
+1. Open [link](https://github.com/apache/pulsar/blob/master/site2/website/data/users.js) in your browser, click “edit” to fork Pulsar repo, create a new branch and edit this file.
+2. Add your company’s information at the end of file. 
+
+Example:
+`
+    {
+        name: 'StreamNative',
+        url: 'https://streamnative.io/',
+        logo: 'https://streamnative.io/static/images/streamnative-logo-white-bakground.png',
+    }
+`
+If your company logo is in white color, add another line after the logo link in JSON file:
```

Comment:
> **Note**  
The `logo_white` parameter is set to `true` only when your company logo uses a white background.